src/components/ApplicationForm/ApplicationForm.jsx

```diff
@@ -1,235 +1,211 @@
+import {useNavigate} from 'react-router-dom'
 import css from './ApplicationForm.module.css'
 import Button from '../Button/Button'
-import sprite from '../../assets/svg/sprite.svg'
 import {useState, useEffect} from 'react'
 import {useModal} from '../Modal/ModalProvider'
 import {useTranslation} from 'react-i18next'
+import TextComponent from '../FormComponents/TextComponent/TextComponent'
+import SelectComponent from '../FormComponents/SelectComponent.jsx/SelectComponent'
+import RadioComponent from '../FormComponents/RadioComponent/RadioComponent'
+import CheckboxComponent from '../FormComponents/CheckboxComponent/CheckboxComponent'
+import {scrollToAnchor} from '../Scroll'
 
 const ApplicationForm = () => {
   const {t} = useTranslation()
-
-  const {setIsModalVisible, setModalContent} = useModal()
-
+  const {isModalVisible, setModalContent, closeModal, openModal} = useModal()
+  const navigate = useNavigate()
   const [inputs, setInputs] = useState({
     name: '',
     email: '',
     city: '',
     role: '',
+    customRole: '',
     certificate: false,
     permission: false,
-    menu: 'traditional',
```

Comment:
tutaj ci źle usunęło bo jak włączasz pierwszy raz modala to oba radio są niezaznaczone